README.md

```diff
@@ -38,6 +38,13 @@ The project uses the [config](https://www.npmjs.com/package/config) npm package
 
 [(back to top)](#bsi-secvisogram-csaf-validator-service)
 
+## Using in production
+
+1. Install production dependencies using `npm ci --omit=dev` from within the `backend` folder
```

Comment:
I suggest to add script in the `package.json` for that...

Reply:
done